src/apps/lisk/verify_message.py

```diff
@@ -0,0 +1,21 @@
+
+async def lisk_verify_message(ctx, msg):
+    from trezor.crypto.curve import ed25519
+    from .helpers import get_address_from_public_key
+    from trezor import wire
+    from trezor.messages.Success import Success
+    from trezor.messages.FailureType import ProcessError
+    from apps.wallet.verify_message import require_confirm_verify_message
+
+    # Lisk signature can be more than 64 bytes
```

Comment:
Lisk use `js-nacl` library for calculating signature, and for message signing use this method https://github.com/tonyg/js-nacl/blob/334f0587103bd3b7e721b92fc1d2a38a8f23708d/README.md#naclcrypto_signmsgbin-signersecretkey--uint8array and final signature more than 64 bytes, because contains a message body.  trezor-core `d25519.verify` raise an error if a signature is not equal 64 bytes.

Reply:
okay, but the protobuf message has a separate field for message and signature.
ISTM we should either take "signed mesage" straight from NaCl and extract the message from that, instead of using a separate field. (and also generate the same thing)
Or the caller (python-trezor / wallet) should do this job and pass the signature separately.

Definitely not "sig + msg" and "msg" as two separate fields. I mean, I don't see any actual security problem with this, but  it's a weird looking cracks for things to fall through; you're throwing away part of a signature without even looking at what's there.

Reply:
@matejcik I've looked at the upcoming `1.0` version of Lisk source code now and found that they will use `crypto_sign_detached` instead of `crypto_sign` like in current version for message signing, and signature will not have a msg body. My bad, I missed this part. So, I will update this PR and make new for `python-trezor`